node/plugin/config.go

```diff
@@ -66,7 +66,7 @@ var (
 	}
 	QuorumIDListFlag = cli.StringFlag{
 		Name:     "quorum-id-list",
-		Usage:    "Comma separated list of quorum IDs that the node will participate in. There should be at least one quorum ID. This list can contain quorums node is already registered with. If the node opts in to quorums already registered with, it will be a no-op.",
+		Usage:    "Comma separated list of quorum IDs that the node will opt-in or opt-out, depending on the OperationFlag. If OperationFlag is opt-in, all quorums should not have been registered already; if it's opt-out, all quorums should have been registered already",
```

Comment:
`all quorums should not have been registered already` sounds like they can't provide quorums that they're already registered with, which isn't true. They _can_ include quorum IDs they've already opted in

Reply:
I'm inclined to align with smart contract API which fails the call if "register quorums that are already registered" or "deregister quorums that are not currently registered" (@0x0aa0 please correct if this isn't the case).

Reply:
We should actually also make opt-in/opt-out consistent w.r.t. this semantics